daemon/daemon_unix.go

```diff
@@ -407,6 +422,12 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *containertypes.
 			return warnings, fmt.Errorf("Cannot use the --read-only option when user namespaces are enabled.")
 		}
 	}
+	if hostConfig.CgroupParent != "" && daemon.usingSystemd() {
+		// CgroupParent for systemd cgroup should be named as "xxx.slice"
+		if len(hostConfig.CgroupParent) <= 6 || !strings.HasSuffix(hostConfig.CgroupParent, ".slice") {
```

Comment:
I'm doing this only to cover `--cgroup-parent=.slice`, in this case, `strings.HasSuffix(hostConfig.CgroupParent, ".slice")` will return true but I think this should not be a valid slice.